pandas/core/nanops.py

```diff
@@ -637,7 +619,7 @@ def _maybe_null_out(result, axis, mask):
             else:
                 result = result.astype('f8')
             result[null_mask] = np.nan
-    else:
+    elif result is not tslib.NaT:
```

Comment:
Oh, I see what you are saying, you were suggesting:

```python
elif result.view('i8') == tslib.iNaT
```

It looks like result is still just a timestamp at this point so it will be the `NaT` object. I don't know if this is a guarantee or not.